scripts/travis/release_packages.sh

```diff
@@ -38,9 +40,9 @@ function promote_nightly() {
     init_s3cmd
 
     # Rename the _CHANNEL_ and _CHANNEL-VARIANT_ pending files
```

Comment:
This comment is no longer correct

Reply:
The new comment here would be something like
> # Copy all files for a given channel to the release bucket.

Also, because it's a copy I think that for each new release this logic will copy all preceding releases as well. Maybe we need a second line to delete the files from `BUILD_BUCKET` (or move them to a separate `BUILD_BUCKET/RELEASED/CHANNEL` directory).